src/main/java/com/easypost/model/Shipment.java

```diff
@@ -548,6 +550,44 @@ public void setTaxIdentifiers(final List<TaxIdentifier> taxIdentifiers) {
         this.taxIdentifiers = taxIdentifiers;
     }
 
+    /**
+     * Get the carrier accounts of this shipment.
+     *
+     * Get the carrier accounts list.
+     *
+     * @return List of carrier accounts.
+     */
+    public List<CarrierAccount> getCarrierAccounts() {
+        return carrierAccounts;
+    }
+
+    /**
+     * Set the carrier accounts list.
+     *
+     * @param carrierAccounts the list of carrier accounts.
+     */
+    public void setCarriersAccounts(final List<CarrierAccount> carrierAccounts) {
+        this.carrierAccounts = carrierAccounts;
+    }
+
+    /**
+     * Get the service used for one-call buys.
```

Comment:
Having a `service` present on a shipment is not indicative of a `one-call-buy` and thus this should be removed as it could be misleading. A service will become present when a shipment is purchased regardless if they used one call or two (and rated the shipment).

Please do this for all 4 of the references here.

```suggestion
     * Get the service used.
```